src/log_outgoing_requests/handlers.py

```diff
@@ -14,11 +14,17 @@ def emit(self, record):
 
             # save only the requests coming from the library requests
             if record and record.getMessage() == "Outgoing request":
+                exclude_headers = ["Authorization"]
+                safe_req_headers = {
+                    k: v
+                    for k, v in record.req.headers.items()
+                    if k not in exclude_headers
+                }
+
```

Comment:
Maybe it'd be better to keep the exclude_headers's name but replace the value with something like `***hidden***`? Now we can't see in the logs if the request had Authorization at all.